x-pack/plugins/apm/public/hooks/use_transaction_latency_chart_fetcher.ts

```diff
@@ -25,9 +26,21 @@ export function useTransactionLatencyChartsFetcher() {
       end,
       transactionName,
       latencyAggregationType,
+      comparisonType,
     },
   } = useUrlParams();
 
+  const {
+    comparisonStart = undefined,
+    comparisonEnd = undefined,
```

Comment:
You could define a type here rather than using `undefined` as a default initial value `{ comparisonStart, comparisonEnd }: Partial<Record<string, string>>`

Reply:
@ogupte I fixed this issue in another PR 😰 https://github.com/cauemarcondes/kibana/blob/apm-time-comparison-transactions-table/x-pack/plugins/apm/public/components/shared/time_comparison/get_time_range_comparison.ts#L50:L51, once it is merged I'll update this code.